### PR TITLE
[EDMTP-435] Added ExpressionInjector.

### PR DESCRIPTION
**Pull request by mhoroshko**

### CODE REVIEW

**Comment by mhoroshko** [comment]
I have read the CLA Document and I hereby sign the CLA

File: core/src/test/java/com/exadel/aem/toolkit/api/annotations/injectors/ExpressionInjectorIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+import io.wcm.testing.mock.aem.junit.AemContext;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ExpressionInjectorIntegrationTest {
```
Comment (smiakchilo) [inline core/src/test/java/com/exadel/aem/toolkit/api/annotations/injectors/ExpressionInjectorIntegrationTest.java:43]: ..Integration ?)

Comment (smiakchilo) [reply core/src/test/java/com/exadel/aem/toolkit/api/annotations/injectors/ExpressionInjectorIntegrationTest.java:43]: Generally, this is a great and endeavorous input. Thank you so much!) 

But it raises  a lot of questions and doubt... Basically, this is as expected)) We need to discuss it more

**Review by liubou-masiuk — COMMENTED** [review]
Please check the comments form #235 and #236

**Comment by mhoroshko** [comment]
Already implemented.

**Pull request closed without merging**